modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -74,21 +77,28 @@ const EditorUpload = function (editor: Editor): EditorUpload {
     return content;
   };
 
-  const replaceImageUrl = function (content: string, targetUrl: string, replacementUrl: string): string {
-    content = replaceString(content, 'src="' + targetUrl + '"', 'src="' + replacementUrl + '"');
+  const replaceImageUrl = (content: string, targetUrl: string, replacementUrl: string, usePlaceholder?: boolean): string => {
+    if (usePlaceholder) {
```

Comment:
This should not be used here... this is for the placeholder text string that gets added to the body, it's not for images.